src/Panel/Routes/Routes.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Kirby\Panel\Routes;
+
+use Closure;
+
+/**
+ * @package   Kirby Panel
+ * @author    Bastian Allgeier <bastian@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://getkirby.com/license
+ * @since     6.0.0
+ */
+abstract class Routes
+{
+	protected static string $prefix = '/';
+	protected static string $type = 'route';
+
+	public function __construct(
+		protected array $area,
```

Comment:
Not all routes are from an area, e.g. routes from fields etc. - and since we want to de-emphasize areas in the long run and we only use it for the id in this class, I wonder whether we should just ask for

```suggestion
		protected string $areaId,
```